src/main/java/com/github/technus/tectech/loader/thing/MachineLoader.java

```diff
@@ -2234,6 +2236,9 @@ public void run() {
                 new GT_MetaTileEntity_DebugStructureWriter(15499, "debug.tt.writer", "Debug Structure Writer", 14)
                         .getStackForm(1L));
         UnusedStuff.set(new ItemStack(Blocks.air));
+        hatch_CreativeUncertainty.set(
+                new GT_MetaTileEntity_Hatch_CreativeUncertainty(15510, "debug.tt.certain", "Uncertainty Resolution", 14)
```

Comment:
Please put “debug” or “creative” in the localized name to make it easy to search.

Reply:
Creative maintenance has no "creative" in the name too isn't it? Can easily search for it by "uncertainty" so it's enough imo

Reply:
Why not make this block craftable in endgame? that's when multiple quantum computers are needed and the uncertainty minigame is not really fun after doing it for the n-th time
In that case the name would be fine